webserver/static/gulpfile.js

```diff
@@ -113,13 +112,22 @@ function buildScripts() {
   ]).then(writeManifest);
 }
 
-gulp.task('styles', buildStyles);
+gulp.task('styles', function () {
+  return buildStyles(writeManifest);
+});
 gulp.task('scripts', buildScripts);
 
 gulp.task('watch', ['styles', 'scripts'], function () {
   let watch = require('gulp-watch');
 
-  watch(path.resolve(STATIC_DIR, '**/*.less'), buildStyles);
+  watch(path.resolve(STATIC_DIR, '**/*.less'), function () {
```

Comment:
I'm pretty sure you can provide a gulp task as a callback to watch, instead of a function, so by setting it to `['styles']` you should get the normal gulp output without having to reproduce it in another callback